researchstudio-sat/webofneeds
# Fix  remote connect

- Connect and SocketHint messages from other Atoms show up without a reload again
- Fixes #3145 -> usability tagsinput
- Fixes #3178 -> suggest atom viewer loading
- Fixes #3123 -> just moved stuff from one file to another and changed the imports (scss only)
- Fixes #3195 -> create atom within a persona
How To Test:
- Connect: connect with debugBot, and write connect -> new connection should be visible without reload
- Hints: create a search job atom-> and see how the jobBot hints roll in
- #3145: go to the create view of a custom post and add exactly one tag (without enter or , ) leave the input field with tab or by clicking something else -> see that the tag has been added anyway
- #3178: Create a direct response atom to any other atom -> copy the share link of the created direct response atom -> logout -> open the copied url -> reload -> you should see that the atom in response is loaded (this was not the case before)
- #3195 -> see description of issue
